provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -335,7 +335,7 @@ def _get_metadata_device(self,
                              cvg: int) -> Text:
         store = self.provider
         metadata_device = Text(store.get(
-            f'node>{machine_id}>storage>cvg[{cvg}]>devices>metadata'))
+            f'node>{machine_id}>storage>cvg[{cvg}]>devices>metadata')[0])
```

Comment:
Please add some documentation with respect to this change,
```
+        # conf-store returns a list of devices, thus, the function
+        # must return a single metadata device path instead of a string of
+        # list.
```

Reply:
Done.